app/public/locales/pt-BR/login.json

```diff
@@ -0,0 +1,14 @@
+{
+  "welcome": "Bem-Vindo",
+  "chooseSignIn": "Por favor escolha tua forma preferida para acessar.",
+  "generatingQrCode": "Gerando QrCode...",
+  "scanQrCode": "Escaneie este QrCode ou copie + cole em tua carteira. Ou clique para logar com tua carteira do navegador.",
+  "clickToConnect": "Clique para conectar",
+  "compatibleWallets": "Carteiras compatíveis",
+  "recommendedWallet": "Carteira recomendada",
+  "otherWallets": "Outras carteiras",
+  "install": "Instalar",
+  "open": "Abrir",
+  "use": "Use",
+  "instead": ""
```

Comment:
Instead is an empty string, is this intentional?

Reply:
Yes, in portuguese we dont use any expression in the end of that phrase